apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -266,6 +266,19 @@ public function enqueue_script_and_style() {
 	 * Register blocks.
 	 */
 	public function register_blocks() {
+		register_block_type(
+			'a8c/navigation-placeholder',
```

Comment:
Maybe `a8c/navigation-menu`? This mirrors the potential naming in the core project. Depending on what our needs are this block may later be editable.

Reply:
Done ✅